src/mqueryfront/src/SearchJobs.js

```diff
@@ -1,154 +1,174 @@
-import React, { Component } from "react";
+import React from "react";
 import { Link } from "react-router-dom";
-import ErrorBoundary from "./ErrorBoundary";
-import axios from "axios";
-import { API_URL } from "./config";
 import PriorityIcon from "./components/PriorityIcon";
 import ActionClose from "./components/ActionClose";
 import ActionCancel from "./components/ActionCancel";
 import StatusProgress from "./components/StatusProgress";
-
-class SearchJobRow extends Component {
-    constructor(props) {
-        super(props);
-
-        this.state = {
-            cancelled: false,
-        };
-
-        this.handleCancelJob = this.handleCancelJob.bind(this);
-    }
-
-    handleCancelJob() {
-        axios.delete(API_URL + "/job/" + this.props.id).then((response) => {
-            this.setState({ cancelled: true });
-        });
-    }
-
-    render() {
-        const submittedDate = new Date(
-            this.props.submitted * 1000
-        ).toISOString();
-
-        let status;
-        if (this.state.cancelled) {
-            status = "cancelled";
-        } else {
-            status = this.props.status;
-        }
-
-        let actionBtn;
-        if (
-            status === "cancelled" ||
-            status === "expired" ||
-            status === "done"
-        ) {
-            actionBtn = <ActionClose onClick={this.props.onClose} />;
-        } else {
-            actionBtn = <ActionCancel onClick={this.handleCancelJob} />;
-        }
-
-        let rule_author = this.props.rule_author
-            ? this.props.rule_author
-            : "(no author)";
-
-        return (
-            <tr>
-                <td>
-                    <div className="d-flex">
-                        <div
-                            className="text-truncate"
-                            style={{ minWidth: 50, maxWidth: "20vw" }}
+import FilteringTableHeader from "./components/FilteringTableHeader";
+import FilteringTitle from "./components/FilteringTitle";
+import Pagination from "react-js-pagination";
+import PropTypes from "prop-types";
+import { finishedStatuses } from "./QueryUtils";
+
+const SearchJobRowEmpty = () => {
+    return (
+        <tr>
+            <td>
+                <div className="d-flex">
+                    <span className="invisible">sth</span>
+                </div>
+                <p className="mb-0 invisible" style={{ fontSize: 11 }}>
+                    sth
```

Comment:
Looks like accident, I'd change to something else like:

```suggestion
                    &nbsp;
```

Reply:
OK. Invisible character in invisible element. Why not?